source/languages/en/dataplatform/using-dataplatform/configuration/getting-started-with-cache-proxy.md

```diff
@@ -221,7 +221,7 @@ telnet $CACHE_PROXY_HOST $CACHE_PROXY_STATISTICS_PORT
 
 Returns statistic results:
 
-```javascript
+```Json
```

Comment:
I didn't hear back, so I changed it myself: https://github.com/basho/basho_docs/commit/1d6fc80bc346670d95fbd88c0666d1379275f5bb